### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by srini1978**
Removed the word API and replaced it with SDK

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,14 @@ The carbonQL acts like a facade, it might call out to other APIs, CSV files, DBs
 
 It provides a common interface to all the various models, makes opinionated decisions about which model to use and how its results should be transformed into the format you need for your calculations. In the above diagram we can see that the carbonQL is intended to connect to various backend carbon datasets like Climatiq, Cloud carbon co-efficients to bring back emissions data given the usage.
 
+The interface definition would be generic i.e irrespective of the underlying data sources (hereafter referred to as models) being used , the method names across all tge data models would be the same. This would provide uniformity to the callers or users to use the carbonQL SDK.  For example some of the methods we have envisioned  in the interface include :
```
Comment (Oleg-Zhymolokhov) [inline README.md:52]: "across all tge data" - there's a typo. Should be "the"?

File: README.md [hunk 2]
```diff
@@ -6,19 +6,19 @@ The friction to measuring software emissions isn't that we need to know how, it'
 
 ## Solution
 
-Therefore, the intent of this project is to build a single API codenamed *carbonQL* that you can use to measure your software emissions for every runtime environment. If your application runs on bare-metal servers in the private cloud, virtualized servers in the public cloud, mobile, desktop, and laptop end-user devices, the carbonQL API  gives you data for all of them.
+Therefore, the intent of this project is to build a SDK or a framework codenamed *carbonQL* that you can use to measure your software emissions for every runtime environment. If your application runs on bare-metal servers in the private cloud, virtualized servers in the public cloud, mobile, desktop, and laptop end-user devices, the carbonQL SDK  gives you data for all of them.
```
Comment (Oleg-Zhymolokhov) [inline README.md:9]: "a SDK" -> "an SDK". S is pronounced as "ess".

**Review by Oleg-Zhymolokhov — COMMENTED** [review]
Just some small propositions for spelling

File: README.md (hunk 1 above)
Comment (srini1978) [reply README.md]: @Oleg-Zhymolokhov  Fixed the issue

File: README.md (hunk 2 above)
Comment (srini1978) [reply README.md]: @Oleg-Zhymolokhov  As per latest understanding, removed all references to the word SDK and replaced it with framework

**Pull request merged**